src/site/xdoc/thanks.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.    
+-->
+<document>
+
+ <properties>
+  <title>Apache James Server 3 - Guice configuration</title>
+ </properties>
+
+<body>
+
+  <section name="Thanks">
+
+    <p>James is a successful project because of the large and diverse community that contributes to it.</p>
+    <p>Here is a list of companies which actively contribute to James by contributing in the evolution of James or by giving some resources/services.</p>
+
+    <subsection name="Companies description">
+    <table>
+
+        <tr>
+          <th width="30%">Company</th>
+          <th>Description</th>
+        </tr>
+
+        <tr>
+          <td><img src="images/linagora.png" /></td>
+          <td><p>We are an Open Source Software Company.</p>
```

Comment:
"OSS company" is a broad term.
Being referred to as an ISV doing only OSS projects might be closer to what linagora actually is ?
We might also include references to products we're "shipping" (OBM and co.), if appropriate.